tests/auth/test_backends.py

```diff
@@ -26,35 +25,67 @@ def test_get_user(username, email, user, exp_exception):
         assert test_user == found_user
 
 
-@pytest.mark.parametrize(('result', 'exp_email', 'exp_exception'),
+@pytest.mark.parametrize(('result', 'exp_email'),
                          [({'scopes': ['assume:mozilla-user:biped@mozilla.com',
                                        'assume:mozillians-user:biped'],
                             'clientId': 'mozilla-ldap/biped@mozilla.com'},
-                           'biped@mozilla.com',
-                           False),
+                           'biped@mozilla.com'),
                           ({'scopes': ['assume:mozillians-user:biped'],
                             'clientId': 'mozilla-ldap/biped@mozilla.com'},
-                           None,
-                           True),
-                          # typical mozillians account logged in with Auth0
+                           "biped@mozilla.com"),
```

Comment:
This testcase is a duplicate of the previous one (the scopes aren't used).

I think some extra testcases for the client ID splitting would be useful, eg:
`foo/mozilla-ldap/fake@email.com`
`mozilla-ldap/` (no trailing email)
`foo`

etc